examples/src/main/java/org/apache/spark/examples/ml/JavaModelSelectionViaTrainValidationSplitExample.java

```diff
@@ -33,6 +33,16 @@
 
 /**
  * Java example for Model Selection via Train Validation Split.
+ *
+ * A simple example demonstrating model selection using TrainValidationSplit.
```

Comment:
We can consolidate this comment line with the one above, this one is a bit redundant. Maybe something like `Java example demonstrating model selection using TrainValidationSplit"